samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <MauiVersion>6.0.312</MauiVersion>
```

Comment:
What do you think about __not__ adding this to the Sample App?

I'd have to test this theory, but I  think it would allow us to ensure that the Sample App continues to work with newer versions of .NET MAUI.

I.e. If I have .NET MAUI v6.0.400 installed on my machine, it would build + run the sample app using v6.0.400, right?

Maybe we could also add this scenario to the CI workflow to ensure that newer versions of .NET MAUI successfully compile and are still compatible with `CommunityToolkit.Maui`?

Reply:
Well, this is used in case you have multiple workloads installed on your device. 
On the one hand we can remove it at all because we set specific MAUI version on ci. 
On the other hand if we use different versions for development and testing we may receive unexpected behavior.

Reply:
Yup! My thought is that by removing it we can test newer versions of MAUI with the Sample App and then we'll know if the .NET MAUI Community Toolkit is working with the newest version of .NET MAUI instead of relying on a user to open an Issue.

Reply:
I've just updated MAUI to 6.0.400 and was not able to build the project. Seems like this tag pins .NET MAUI version and looks for exactly this version installed.
But because we pin it on yaml, we should not break development on a local devices. If build fails, we'll need to upgrade .NET MAUI